performance/src/main/kotlin/org/jetbrains/ring/BenchmarkResult.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jetbrains.ring
+
+class BenchmarkResult(val name: String, val status: Status,
+                      val score: Double, val runtimeInUs: Double,
+                      val repeat: Int, val warmup: Int) {
+    enum class Status(val value: String) {
+        PASSED("PASSED"),
+        FAILED("FAILED")
+    }
+
+    fun toJson(): String {
```

Comment:
I a bit disagree in this point. In my opinion, it's resposibility of entity to know how it should be converted to another representation.

Reply:
I won't insist because in this trivial case it's fine, but in more complicated cases it will lead to bloated code with unnecessary cyclic dependencies between classes.